src/main/java/io/jenkins/plugins/jfrog/JfStep.java

```diff
@@ -220,7 +230,40 @@ private void addConfigArguments(ArgumentListBuilder builder, JFrogPlatformInstan
         builder.add("--interactive=false");
         // The installation process takes place more than once per build, so we will configure the same server ID several times.
         builder.add("--overwrite=true");
+    }
+
+    /**
+     * Add build-info Action if the command is 'jf rt bp' or 'jf rt build-publish'.
+     *
+     * @param run              - The Jenkins project
+     * @param taskOutputStream - Task's output stream
+     * @throws JsonProcessingException in case of any unexpected error during the JSON parsing of the build info URL.
+     */
+    private void addBuildInfoActionIfNeeded(Run<?, ?> run, ByteArrayOutputStream taskOutputStream) throws JsonProcessingException {
```

Comment:
Let's add a test for this method, or to the output extraction logic of this method.